app/jobs/models.py

```diff
@@ -0,0 +1,91 @@
+from datetime import timedelta
+
+from django.conf import settings
+from django.db import models
+from django.urls import reverse
+from django.utils import timezone
+from django.utils.translation import ugettext_lazy as _
+
+# Create your models here.
+
+JOB_TYPE_CHOICES = (
```

Comment:
This should be defined on the model itself so `Job.JOB_TYPES` can be referenced throughout.  Likewise, it would be helpful if you provided explicitly defined entries for each given job type, so when someone wants to query for the specific type, they're not specifying the type string in line.